apps/avatax/scripts/run-webhooks-migration.ts

```diff
@@ -0,0 +1,104 @@
+import * as dotenv from "dotenv";
+
+import { SaleorCloudAPL } from "@saleor/app-sdk/APL";
+import { otelSdk } from "@saleor/apps-otel/src/instrumentation";
+import { WebhookMigrationRunner } from "@saleor/webhook-utils";
+
+import { ObservabilityAttributes } from "@saleor/apps-otel/src/lib/observability-attributes";
+import { createInstrumentedGraphqlClient } from "../src/lib/create-instrumented-graphql-client";
+import { loggerContext } from "../src/logger-context";
+import { appWebhooks } from "../webhooks";
+import { createLogger } from "./logger";
+
+dotenv.config();
```

Comment:
This line will evaluate AFTER imported files. It can lead to heisenbugs. You should run config() in L2 or even in L1

`require('dotenv').config()`

Or
https://www.npmjs.com/package/dotenv#preload

Reply:
Done with preload method